vexuser.c

```diff
@@ -169,5 +191,32 @@ vexOperator( void *arg )
 	return (msg_t)0;
 }
 
+//spins the intake/outake wheels
+void spinWheels(int speed)
+{
+		vexMotorSet(LeftIntakeWheel, speed);
+		vexMotorSet(RightIntakeWheel, speed);
+}
+
+//sets the speed of the intake arms
+void moveIntakeArms(int speed)
+{
+	vexMotorSet(LeftIntakeArm, speed);
+	vexMotorSet(RightIntakeArm, speed);
```

Comment:
The two intake motors need to be rotating in opposite directions since the motors are facing opposite directions.